app/templates/index.hbs

```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid">
+    <div class="ui stackable three column grid filtered">
```

Comment:
The bug is in event card, not in upcoming section. This is the 5th time I'm repeating this

Reply:
i apologize sir. i thought that the event card is under upcoming section so i was trying to make it in those cards only.